plans/future/platform/kb-artifact-migration/in-progress/KBAR-0230/KBAR-0230.md

```
@@ -0,0 +1,280 @@
+**Estimated hours:** 3
+
+Write Vitest unit tests in `__tests__/generateStoriesIndex.test.ts`. Cover TC-01 through TC-12 as specified in `_pm/test-plan.yaml`. Mock Drizzle DB client with `vi.mock`. Include snapshot test for format regression guarding. Verify round-trip compatibility via `parseStoriesIndex()`.
+
+---
+
+## QA Notes
+
+```yaml
+qa_notes:
+  verdict: CONDITIONAL_PASS
+  decided_at: "2026-03-03T12:05:00Z"
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)